### DISCUSSION TITLE
Django integration doc

### DISCUSSION THREAD

**Discussion by sureshvv** [mdbootstrap/TW-Elements#1695] [Support from community] [answered]
I already have integrated tailwind css into my Django app from a previous project.

I am now looking to integrate TE from thiis doc:

    https://tailwind-elements.com/docs/standard/integrations/django-integration/

In which step is tailwind-elements being installed? 

Is this implicitly done in:

    npm install -D tailwindcss postcss autoprefixer
    npx tailwindcss init -p

Or is an extra step missing in the doc?

**Comment by juujisai** [comment, marked as answer]
Hi! I guess this step is missing. Try installing the `tw-elements` package after `tailwindcss`
`npm install tw-elements
`
Best Regards!
